.jest/setup.js

```diff
@@ -0,0 +1,10 @@
+import { cleanUpTestSpaces } from '@contentful/integration-test-utils'
+import { initConfig } from '../test/contentful-config'
+
+beforeAll(() => {
+  return initConfig()
```

Comment:
This tests set the actual file, it should be a better bet than mocks for e2e testing (that way we're sure the file parsers works)

Reply:
Hmm yeah I think I agree with that, but at the same time I don't think tests should pollute my system by writing something to it, at least without cleaning it up.

Reply:
Do you know where the config is written? ideal this stuff ends up in a user's ` tmp` [directory](https://www.geeksforgeeks.org/node-js-os-tmpdir-method/).  I think jest has some tooling for that 🤔

Reply:
This will write the file only if it doesn't exist. if we clean up it will also pollute your system by removing the file completely. Unless, we create a different config file for testing, but again that means we're manipulating the actual behaviour of the CLI 🤷🏼‍♂️

Reply:
@marcolink `initConfig` adds the file to home directory `~` saving it in tmp means we have to tell the program to look for it there, right?

Reply:
That's true! 
Which might be ok, as we don't want a test to overwrite our local root config, right?

Reply:
My bad, we actually have a way to override the config path `CONTENTFUL_CONFIG_FILE`. 
the way you both described means we'll be testing more functionality then. on it...

Reply:
addressed 👍🏼